stock_archive_constraint/tests/test_location_archive_constraint.py

```diff
@@ -9,6 +9,9 @@ class TestLocationArchiveConstraint(TransactionCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        cls.env = cls.env(
+            context=dict(cls.env.context, test_stock_archive_constraint=True)
```

Comment:
Wouldn't this go in the conflicting module test?

Reply:
This one is the module

Reply:
No, it's for testing this functionality. The absence of this context is what makes that the constraints are not raised for the tests of the other modules.

Reply:
Nevermind. I got it already